README.md

```diff
@@ -5,10 +5,10 @@ Provisioning frontend service for `cloud.redhat.com`
 ## Development
 This project is based on react and uses the `cloud.redhat.com` micro-frontend architecture
 
-[patternfly](https://www.patternfly.org/v4/) - an opinionated UX/UI library for enterprise applications
-[insights-chrome](https://github.com/RedHatInsights/insights-chrome) - a wrapper that provides an api for the global application (menus / auth / etc)
-[react-query](https://react-query.tanstack.com) - Manages the server state
-[react-tracked](https://react-tracked.js.org) - Optimizes re-renders for a global state management
+[patternfly](https://www.patternfly.org/v4/) - an opinionated UX/UI library for enterprise applications </br>
+[insights-chrome](https://github.com/RedHatInsights/insights-chrome) - a wrapper that provides an api for the global application (menus / auth / etc) </br>
+[react-query](https://react-query.tanstack.com) - Manages the server state </br>
```

Comment:
nitpick: I don't think `</br>` is good use here :thinking:

Reply:
why not?

Reply:
Why not use a Markdown native utils for it, like proper list for example?

Reply:
Is there a decision? 😺

Reply:
I still disagree with this, but it's so minor, I'll not block on it :)

Reply:
@amirfefer you've said offline you will use the list, shall we wait for that change?